nemo/export/tensorrt_llm.py

```diff
@@ -83,6 +83,17 @@ def wrapper(*args, **kwargs):
     use_pytriton = False
 
 
+def determine_quantization_settings(
```

Comment:
Could you add docstrings for all new / edited functions?

Reply:
(same remark for typing)

Reply:
If possible let's please use docstring convention from https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/llm/api.py:

So sth like:
```
One or two sentences what's going on.

Args:
    param1 (Type): It is what it is.
    param2 (Type): It is what it is.

Returns:
    Type: It is what it is.
```